gef-ui/server/utils/helpers.js

```diff
@@ -174,8 +174,8 @@ const generateActionsArrayForItem = ({ href, visuallyHiddenText, text, id }) =>
 const previewItemConditions = (previewParams) => {
   const { issuedHref, unissuedHref, issuedToUnissuedHref, shouldDisplayChangeLinkIfIssued, shouldDisplayChangeLinkIfUnissued, item, app } = previewParams;
   let summaryItems = [];
-  const statusMIA = [CONSTANTS.DEAL_STATUS.READY_FOR_APPROVAL, CONSTANTS.DEAL_STATUS.SUBMITTED_TO_UKEF];
-  const statusAIN = [...statusMIA, CONSTANTS.DEAL_STATUS.UKEF_ACKNOWLEDGED, CONSTANTS.DEAL_STATUS.SUBMITTED_TO_UKEF, CONSTANTS.DEAL_STATUS.CHANGES_REQUIRED];
+  const statusMIA = [DEAL_STATUS.READY_FOR_APPROVAL, DEAL_STATUS.SUBMITTED_TO_UKEF];
+  const statusAIN = [...statusMIA, DEAL_STATUS.UKEF_ACKNOWLEDGED, DEAL_STATUS.SUBMITTED_TO_UKEF, DEAL_STATUS.CHANGES_REQUIRED, DEAL_STATUS.CANCELLED];
```

Comment:
I'm not entirely certain how this works, but it looks like when the deal is cancelled line 225 was being triggered. It looks like `validStatus` checks if the status is not one of these statuses
```
  } else if (ukefDecisionAccepted && item.id === 'coverStartDate' && validStatus) {
    summaryItems = generateActionsArrayForItem({
      href: issuedHref,
      visuallyHiddenText: item.label,
      text: ukefDecisionAccepted && ACTION_TEXT.CHANGE,
      id: item.id,
    });
  }
```

Reply:
Would love to completely refactor these files at some point...